http/templates/bracket_detail.html.tera

```diff
@@ -43,7 +43,7 @@
                     <td class="{% if race.player_2.winner %} winner-cell {% elif race.player_2.loser %} loser-cell {% endif %}">
                         <span>{{ macros::player_detail(name=race.player_2.name) }}</span>
 
-                        {% if race.player_1.status %}
```

Comment:
whoops! luckily it's logically impossible (and thus practically pretty unlikely) for there to be a race with player 1's status set and player 2's unset. good catch!